cmd/ui/src/views/Explore/EntityInfo/EntityInfoContent.tsx

```diff
@@ -15,36 +15,22 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Box } from '@mui/material';
-import { allSections, entityInformationEndpoints, EntityKinds } from 'bh-shared-ui';
+import { EntityKinds } from 'bh-shared-ui';
 import React from 'react';
-import { useQuery } from 'react-query';
 import EntityInfoDataTableList from './EntityInfoDataTableList';
 import EntityObjectInformation from './EntityObjectInformation';
 
 export interface EntityInfoContentProps {
     id: string;
-    nodeType: EntityKinds;
+    nodeType: EntityKinds | string;
```

Comment:
String type was added here in anticipation of someone creating their own node with whatever label they want to add to it. Technically we only return `Unknown` from the API for entities with labels not in our schema but to me it seems like it would make sense to start surfacing that type instead of blanket returning `Unknown` in which case `string` for these props would serve as a catchall.

Your suggestion does fit best with what is currently in place. Maybe this is an overly eager update; What do you think?

Reply:
Thanks for the details! I think the catchall makes sense and is a good idea! Especially considering vizsla in the future. 

We may need to consider adding some type guards in the future instead of casting as EntityKinds (looking at `EntityInfoDataTableList.tsx`) to handle when nodeType is a string! I dont think we need to make any changes rn, but when we start allowing outside EntityKinds, we can search for instances of `as EntityKind`!

Reply:
I made an attempt to tighten up the type validation here. It's not perfect but a little better than before. We may need to get an overall picture of where all this is relevant to properly clean this up. I appreciate your push for making this better